ci-operator/step-registry/baremetalds/assisted/gather/baremetalds-assisted-gather-commands.sh

```diff
@@ -45,6 +45,7 @@ sos report --batch --tmp-dir /tmp/artifacts \
 
 # TODO: remove when https://github.com/sosreport/sos/pull/2594 is available
 cp -v -r /var/lib/libvirt/dnsmasq /tmp/artifacts/libvirt-dnsmasq
+cp -v -r /var/log/swtpm/libvirt/qemu /tmp/artifacts/libvirt-qemu || true
```

Comment:
it seems like the ``TODO`` refers to the two lines
maybe it's better to separate between them?